### PR TITLE
Table frint

### PR DESCRIPTION
**Pull request by AlexValder**

### CODE REVIEW

File: DBLab2/DBLab2/Common/Printer.cs [hunk 1]
```diff
@@ -61,6 +61,46 @@ public static void Debug(string template, params object[] args)
...
+            }
+
+            var width = new List<int>();
+            for (int i = 0; i < columns; ++i) {
+                width.Add(0);
+                for (int j = 0; j < rows; ++j) {
+                    if (width[i] < data[j][i].Length) {
+                        width[i] = data[j][i].Length;
+                    }
+                }
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/Printer.cs:78]: `width.Add(data.Max(arg => arg[1].Length))`

Comment (ArthurMamedov) [reply DBLab2/DBLab2/Common/Printer.cs:78]: Loops are shit...

Comment (AlexValder) [reply DBLab2/DBLab2/Common/Printer.cs:78]: Okay...

**Review by ArthurMamedov — APPROVED** [review]
Nice!

**Pull request merged**